src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
apply object destructing 
```suggestion
export const Person = ({ person }) => {
  const {...} = person;
  
  return (
    <section className="Person">
```

Reply:
You need to use return statement:

  const {
    name,
    age,
    sex,
    isMarried,
    partnerName,
  } = person;

  return (
    Your jSX here
  )